src/config/default_config.yaml

```diff
@@ -10,7 +10,7 @@ events: true
 # history: experimental support
 history: true
 repl: true
-traceSharingEnabled: true
+traceSharingEnabled: false
```

Comment:
Please, update it in both copies of the config file: config/ and src/config/. we need to move all usages to src/config but this is not part of this PR

Reply:
(i was told it's already `false` there: sorry)